app/models/spree/webinar_registration.rb

```diff
@@ -20,12 +20,18 @@ def sync_with_gotomeeting
     data = SpreeGotomeeting.client.post(
```

Comment:
I would call this variable `response` now, since it isn't the data itself anymore now that you've removed `parse`. Then you can continue to use `data` for the parsed data.